app/index.js

```diff
@@ -84,6 +83,10 @@ module.exports = class extends Generator {
     this.fs.writeJSON(this.destinationPath('package.json'), pkg);
   }
 
+  conflicts() {
+    this.fs.append(this.destinationPath('.eslintignore'), 'generators/*/templates\n');
+  }
```

Comment:
It feels kinda weird to handle this in the `conflicts` phase. But the file from generator-node isn’t available in `writing` phase.